### PR TITLE
Get and set comments for each movie.

### LINKED ISSUE #8: [4pt]  - Create feature: add new comment - Student B
Implement the "add new comment" form on the Comments pop up according to the wireframe:

Screenshot 2021-06-27 at 22 32 36

When the user clicks on the "Comment" button, the data is recorded in the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) and the screen is updated.

### LINKED ISSUE #11: [3pt]  - Display comments for a given item on the Comments pop-up - Student B
When the popup loads, the webapp retrieves data from the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item comments.

Screenshot 2021-06-27 at 22 21 00

This task **does not** include the counter of the comments.

### LINKED ISSUE #14: [4pt]  -Display Comments pop up with selected item's details - Student B
Create a comments popup only with the top section (displaying details of the selected item)

Screenshot 2021-06-27 at 22 10 45

When the popup loads, the webapp retrieves data from the selected API and shows details about the selected item.

Add event to button prepared by your teammate that will open your popup. You need to communicate with your team member to make sure that you do not block each other (most likely the easiest solution is to merge the list of items feature to the dev branch before you try to add that event).

### PR DESCRIPTION
**Pull request by jssol**
- Get comments from the involvement API
- Send comments to the API

### CODE REVIEW

File: src/modules/involvement.api.js [hunk 1]
```diff
@@ -20,17 +20,17 @@ const setComment = async (movieID, comment) => {
...
+    item_id: movieID,
+    username: getUser(),
+    comment,
+  };
   const appID = 'YUqI88f5a8VwBEjald5b';
   const res = await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${appID}/comments`, {
     method: 'POST',
-    body: JSON.stringify(data)
+    body: JSON.stringify(data),
   });
```
Comment (Ismailco) [inline src/modules/involvement.api.js]: you need to add the content type
```
body: JSON.stringify(data),
headers: {
      'Content-type': 'application/json; charset=UTF-8',
    },
```

**Review by Ismailco — APPROVED** [review]
Hi, @jssol your code is Nicely done 👍🏼
Just add the content type before you POST to the API

File: src/modules/involvement.api.js (hunk 1 above)
Comment (jssol) [reply src/modules/involvement.api.js]: Thanks for the tip :100:
I have added the headers, now we can add new comments

**Review by Ismailco — APPROVED** [review]
🥇 Very Good

**Comment by Ismailco** [comment]
⚠️  You have some issues with the input and textarea

**Comment by jssol** [comment]
> warning You have some issues with the input and textarea

I have fixed the issue

**Pull request merged**